earthpy/plot.py

```diff
@@ -386,14 +408,16 @@ def draw_legend(im_ax, titles=None, cmap=None, classes=None, bbox=(1.05, 1)):
 
     Parameters
     ----------
-    im : matplotlib image object created using imshow()
+    im : matplotlib image object
         This is the image returned from a call to imshow().
-    classes : list (optional)
-        A list of unique values found in the numpy array that you wish to plot.
     titles : list (optional)
         A list of a title or category for each unique value in your raster.
         This is the label that will go next to each box in your legend. If
         nothing is provided, a generic "Category x" will be populated.
+    cmap : str (optional)
```

Comment:
Similar to the above - please ensure that a parameter with a fixed value unless defined is NOT optional but something like titles which has a default set to None is optional! In this case bbox has a fixed value. With that said let's move bbox so be the SECOND parameter in this list! it's common to have parameters with default values FIRST and then optional parameters second.